arbnode/dataposter/dataposter_test.go

```diff
@@ -152,46 +153,36 @@ func TestMaxFeeCapFormulaCalculation(t *testing.T) {
 	}
 }
 
-type stubL1Client struct {
+type stubL1ClientInterface struct {
```

Comment:
Ahh i agree, the reason i named it like this is because we can't really name it a `stubL1Client` as it is not a `ethclient.Client` itself but instead the underlying `rpc.ClientInterface`, i'll rename it to something like `stubL1ClientInner` to make it clearer.